edurange_refactored/user/views.py

```diff
@@ -261,23 +263,39 @@ def instructor():
 
     return redirect(url_for("dashboard.instructor"))
 
-
 @blueprint.route("/notification", methods=["GET", "POST"])
 @login_required
 def notification():
     if request.method == "POST":
         process_request(request.form)
 
-    notificationList = Notification.query.all()
+    notifications_raw = Notification.query.all()
+    notifications_serialized = []
+
+    for notif in notifications_raw:
+        notif_serialized = {}
+        for key, value in notif.__dict__.items():
```

Comment:
Rather than iterate over the columns of each notification entry and exclude the _sa_instance_state, I think it's more clear if you assign each field directly. as in:
```
        notif_serialized['id'] = notif.id
        notif_serialized['detail'] = notif.detail
        notif_serialized['date'] = notif.date.isoformat()
```
to replace this inner for loop